src/kbmod/reprojection.py

```diff
@@ -1,16 +1,19 @@
 import numpy as np
+import concurrent.futures
 import reproject
 from astropy.nddata import CCDData
 from astropy.wcs import WCS
 
 from kbmod.search import KB_NO_DATA, PSF, ImageStack, LayeredImage, RawImage
 from kbmod.work_unit import WorkUnit
 
+# The number of executors to use in the parallel reprojecting function.
+NUM_EXECUTORS = 8
```

Comment:
A quick drive-by comment, but would it be possible to make the number of executors an optional param to the function?

Reply:
Agreed that this would be better as an optional parameter.